static/css/budget.less

```diff
@@ -11,30 +17,30 @@
 
       li {
           .list_item();
-          
+
           a {
               .list_link();
               .jump-link();
           }
       }
     }
-    
-    &.financial-report a, 
+
+    &.financial-report a,
     &.performance-plan a,
     &.funding-request a span{
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/27b8b15f4808f2bf903e7b9ee719465bf975e2cd